SMA connector package: 142-0701-801

dbrgn opened the pull request:
SMA connector, edge mount

https://www.digikey.ch/product-detail/de/cinch-connectivity-solutions-johnson/142-0701-801/J502-ND/35280

![img](https://tmp.dbrgn.ch/screenshots/20190714004541-nwz3fop4.png)

dbrgn committed:
Add pkgcat "Cinch Connectivity Solutions"

dbrgn committed:
Add pkg "CINCH_142-0701-80X"

ubruhin commented:
I would place the `{{NAME}}` label on the right side too, above the `{{VALUE}}` label. This makes the package much more compact. Otherwise it looks good :+1:

dbrgn commented:
Fixed!

![2019-09-10-094824_532x398_scrot](https://user-images.githubusercontent.com/105168/64594369-2a5db580-d3b0-11e9-8d90-20803f080e2c.png)

I also renamed the package to 142-0701-80X to include both plating variants.
